.eslintrc

```diff
@@ -176,7 +176,8 @@ rules:
 
     brace-style:                   # enforce one true brace style
         [2, "1tbs", { "allowSingleLine": true }]
-    camelcase: 2                   # require camel case names
+    camelcase:                     # require camel case names, except in properties
+        [2, { "properties": "never" }]
```

Comment:
inline-style-prefixer's browser data generator outputs objects with keys like `ios_saf`. This disables the `camelcase` check for object properties. We can't easily add `eslint-disable` rules to those specific files because they're generated.

Reply:
Which files are generated?

Reply:
`src/prefix-data/static.js` and `src/prefix-data/dynamic.js`

Reply:
Is there anything else in `prefix-data`? If not, let's can't we just revert this here and drop a new `src/prefix-data/.eslintrc` that overrides it there?

Reply:
I considered that and would be happy to do it, lemme make the case for why I didn't though:
* We'd need it in `scripts` too (the input object to `generateData` also has underscores)
* Really I find the whole lint rule applying to properties extremely presumptuous. Other people's APIs use underscores (this example, JSON from REST APIs, even React has `unstable_foo` methods that people actually use) and it's lame needing an exception for that every single time. If our code were completely self-contained and didn't interact with anyone else's APIs then it would make sense.

Reply:
OK.